pkg/operator/resource/resourceapply/core.go

```diff
@@ -71,6 +71,29 @@ func ApplyService(client coreclientv1.ServicesGetter, recorder events.Recorder,
 	return actual, true, err
 }
 
+// ApplyPod merges objectmeta, does not worry about anything else
+func ApplyPod(client coreclientv1.PodsGetter, recorder events.Recorder, required *corev1.Pod) (*corev1.Pod, bool, error) {
```

Comment:
@deads2k not sure if we want this here, but I don't see any problem with it.

Reply:
shouldn't this at least be its own commit?